src/modules/StudentOrgsHeroSection/StudentOrgsHeroSection.tsx

```diff
@@ -0,0 +1,43 @@
+import { Button, Typography } from 'components';
```

Comment:
Don't need this new module, can reuse `Header` module. See `governance/index.tsx` for usage